src/CMakeLists.txt

```diff
@@ -31,10 +33,18 @@ generate_export_header(kdenowcore
     EXPORT_MACRO_NAME KDENOWCORE_EXPORT
 )
 
-#target_include_directories(kdenowcore PUBLIC ${CMAKE_CURRENT_BINARY_DIR})
+configure_file(data/org.kde.kdenow.event.service.in ${CMAKE_CURRENT_BINARY_DIR}/org.kde.kdenow.event.service)
```

Comment:
We don't really need to use separate service for each plugin type. Use just one for the daemon which is going to instantiate all the plugins and hence all of them register themselves.